docs/backups.rst

```diff
@@ -1,25 +1,140 @@
-Backups
-=======
+CTS backups
+===========
 
-How backups are performed
--------------------------
+Backups are taken regularly and stored on the Caktus backup server. This document explains how to
+access and use those backups.
 
-The databases are hosted on AWS RDS and backups are run there.
+Getting a backup dump and restoring it locally
+----------------------------------------------
 
-How to check that backups are working
--------------------------------------
+Steps you can do ahead of time:
 
-Use the AWS Console and check the RDS instance's backups.  (TBD: where
-to look for that in the console.)
+* Get access to the Caktus backup server (open a tech support request).
 
-How to recover in the event of a lost server
+When you need to restore a backup:
+
+* Make sure you are in your CTS-IRCDeploy directory, and not in the CTS project directory::
+
+    $ git config --get remote.origin.url
+    git@github.com:theirc/CTS-ircdeploy.git
+
+* List the files in the latest backup directory and find the most recent::
+
+    $ backup_path=/mnt/rsnapshot/cts/daily.0/home/caktus-backup
+    $ ssh caktus-backup@backup.caktus.lan ls $backup_path | tail -1
+    cts_turkey.rescue.org-20170927.bz2
+
+  That will list the latest 'turkey' dumpfile. We'll replace 'turkey' with 'iraq' and 'jordan' to
```

Comment:
There's not actually a "turkey" in the ls command - maybe tweak it to ``ssh caktus-backup@backup.caktus.lan ls $backup_path/cts_turkey* | tail -1`` and then we can run it again with "iraq" and "jordan"?

Reply:
Yeah, turkey just ends up being last because it's alphabetically last. But I'll make it more explicit like you suggest.